catalog/opensearch/catalog-opensearch-source/src/test/java/org/codice/ddf/opensearch/source/OpenSearchParserImplTest.java

```diff
@@ -378,17 +383,21 @@ public void populateNullSearchOptions() {
 
   // {@link OpenSearchParser#populateSpatial(WebClient, SpatialSearch, List)} tests
 
-  @Test(expected = IllegalArgumentException.class)
-  public void populateSpatialGeometry() {
+  @Test
+  public void populateSpatialGeometry() throws ParseException {
+
+    Geometry geometry = new WKTReader().read(WKT_GEOMETRY);
     openSearchParser.populateSpatial(
         webClient,
-        mock(Geometry.class),
+        geometry,
         null,
         null,
         null,
         Arrays.asList(
-            "q,src,mr,start,count,mt,dn,lat,lon,radius,bbox,polygon,dtstart,dtend,dateName,filter,sort"
+            "q,src,mr,start,count,mt,dn,lat,lon,radius,bbox,geometry,polygon,dtstart,dtend,dateName,filter,sort"
```

Comment:
It might be cleaner to pull this array out into it's own constant and also update all the other `openSearchParser.populate*()` calls